unity/Assets/Scripts/ArmAgentController.cs

```diff
@@ -127,37 +149,176 @@ public class ArmAgentController : PhysicsRemoteFPSAgentController {
             }
         }
 
-        public override void MoveAhead(ServerAction action) {
-            throw new InvalidOperationException("When using the arm, please call controller.step(action=\"MoveAgent\", ahead=X, right=0).");
+        public override void MoveAhead(
```

Comment:
Another note for all these move action updates, we need to make sure to update the ManipulaTHOR api to also include these on top of the default `MoveAgent`